src/app/address/controllers/addressConfirm.js

```diff
@@ -63,7 +64,8 @@ class AddressConfirmController extends BaseController {
       buildingNameNumber = address.buildingName || address.buildingNumber;
     }
 
-    return `${buildingNameNumber}<br>${address.streetName},<br>${address.addressLocality},<br>${address.postalCode}<br>`;
+    const text = `${buildingNameNumber}<br>${address.streetName},<br>${address.addressLocality},<br>${address.postalCode}<br>`;
+    return { ...address, text };
```

Comment:
We're returning the formatted text with the address data now rather than just the formatted text of the address.